### ISSUE TITLE
Typo in generated migrations script comment

### ISSUE THREAD

**Issue by andreterron** [drizzle-team/drizzle-kit-mirror#38]
When running `drizzle-kit introspect:pg` we get the following header:

```sql
-- Current sql file was generated after introspecting the database
-- If you want to run this migration please uncomment this code before executing migraitons
```

But there's a typo on the last word: `migraitons` should be `migrations`

**Comment by AndriiSherman** [comment]
Thanks! Will add this to the next release, will mention you as a contributor

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.18.1`

**Comment by andreterron** [comment]
Thank you! Happy to help!
